### ISSUE TITLE
try to run sh build.sh giving error

### ISSUE THREAD

**Issue by 07onkar07** [VLprojects/mediasoup-client-swift#15]
warning: (arm64)  failed to insert symbol '__ZTVN6webrtc12_GLOBAL__N_118BlockProcessorImplE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZTSN6webrtc12_GLOBAL__N_118BlockProcessorImplE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_118BlockProcessorImpl15instance_count_E' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_19kSqrt2SinE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZTVN6webrtc12_GLOBAL__N_115EchoRemoverImplE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_115EchoRemoverImpl15instance_count_E' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_17kMaxErlE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZZN6webrtc14FilterAnalyzer17PreProcessFiltersEN3rtc9ArrayViewIKNSt3__16vectorIfNS3_9allocatorIfEEEELln4711EEEE1h' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_131kPreEchoHistogramDataNotUpdatedE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_114kHErrorInitialE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZTVN6webrtc12_GLOBAL__N_121RenderDelayBufferImplE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZTSN6webrtc12_GLOBAL__N_121RenderDelayBufferImplE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_121RenderDelayBufferImpl15instance_count_E' in the debug map.
warning: (arm64)  failed to insert symbol '__ZTVN6webrtc12_GLOBAL__N_125RenderDelayControllerImplE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZTSN6webrtc12_GLOBAL__N_125RenderDelayControllerImplE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_125RenderDelayControllerImpl15instance_count_E' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_115kBandBoundariesE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_114kMinNoisePowerE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_112kSqrtHanningE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZZNK6webrtc8OouraFft10bitrv2_128EPfE2ip' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtcL9rdft_wk1rE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_123WebRtcAecm_kSqrtHanningE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZZN6webrtc12_GLOBAL__N_118LogClippingMetricsEiE24atomic_histogram_pointer' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtcL15kHistBinCentersE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtcL16kNoiseGmmWeightsE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtcL15kCoeffNumeratorE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_115kLowProbabilityE' in the debug map.
warning: (arm64)  failed to insert symbol '_kWeight' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_116kGainTableAnalogE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_113kGenFuncTableE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_19log_tableE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_123kBlocks160w256FirstHalfE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtcL14kLtrFeatureThrE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZNSt3__1L19piecewise_constructE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZZN7cricket19FilterRtpExtensionsERKNSt3__16vectorIN6webrtc12RtpExtensionENS0_9allocatorIS3_EEEEPFbN4absl11string_viewEEbRKNS2_15FieldTrialsViewEE23kBweExtensionPriorities' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert symbol '__ZZN7cricket23WebRtcVoiceMediaChannel31MaybeCreateDefaultReceiveStreamERKN6webrtc17RtpPacketReceivedEE24atomic_histogram_pointer' in the debug map.
warning: (arm64)  failed to insert symbol '__ZTVN6webrtc12_GLOBAL__N_135VideoDecoderSoftwareFallbackWrapperE' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert symbol '__ZTVN6webrtc12_GLOBAL__N_135VideoEncoderSoftwareFallbackWrapperE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZTSN6webrtc12_GLOBAL__N_135VideoEncoderSoftwareFallbackWrapperE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_127kBaseHeavy3TlRateAllocationE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_123kRtcpAnyExtendedReportsE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_126kIncludeCaptureClockOffsetE' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert symbol '__ZZN6webrtc14UlpfecReceiverD1EvE24atomic_histogram_pointer' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_127kBweBackOffFactorExperimentE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZZN6webrtc20PacketArrivalTimeMap14SetNotReceivedExxE5value' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_114kTimestampToMsE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_115kFieldTrialNameE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZZN6webrtc16BitrateAllocatorD1EvE24atomic_histogram_pointer' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_126kBweThroughputWindowConfigE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZNSt3__1L19piecewise_constructE' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert symbol '__ZZN6webrtc15ProbeController12RequestProbeENS_9TimestampEE24atomic_histogram_pointer' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_117kUmaRampupMetricsE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZZN6webrtc27SendSideBandwidthEstimation9UpdateRttENS_9TimeDeltaENS_9TimestampEE24atomic_histogram_pointer' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_120kBweLossBasedControlE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZTVN6webrtc12_GLOBAL__N_121AudioCodingModuleImplE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZTSN6webrtc12_GLOBAL__N_121AudioCodingModuleImplE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_128kNackTrackerConfigFieldTrialE' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert symbol '__ZZN6webrtc20StatisticsCalculator14EndExpandEventEiE24atomic_histogram_pointer' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_115WebRtcCng_kDbovE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZTVN6webrtc12_GLOBAL__N_115AudioEncoderCngE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZTSN6webrtc12_GLOBAL__N_115AudioEncoderCngE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZNSt3__1L19piecewise_constructE' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12video_coding12_GLOBAL__N_115start_code_h264E' in the debug map.
warning: (arm64)  failed to insert symbol '__ZNSt3__1L19piecewise_constructE' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_114kMaxVideoDelayE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_134kMaxScreenSharingLayerFramerateFpsE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZTVN6webrtc12_GLOBAL__N_135BuiltinVideoBitrateAllocatorFactoryE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZTSN6webrtc12_GLOBAL__N_135BuiltinVideoBitrateAllocatorFactoryE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_112kAccMaxErrorE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZTVN6webrtc12_GLOBAL__N_123FrameCadenceAdapterImplE' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert symbol '__ZTVN6webrtc12_GLOBAL__N_119FrameDumpingDecoderE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZTSN6webrtc12_GLOBAL__N_119FrameDumpingDecoderE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_118kMaxFramesBufferedE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZTVN3rtc16RefCountedObjectIN6webrtc12_GLOBAL__N_125EncodedImageBufferWrapperEEE' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN7cricket12_GLOBAL__N_139kUseLegacySimulcastLayerLimitFieldTrialE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_111kFieldTrialE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZNSt3__1L19piecewise_constructE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_115kFieldTrialNameE' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert symbol '__ZZN6webrtc17VideoRenderFramesD1EvE24atomic_histogram_pointer' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert symbol '__ZZN6webrtc27DefaultOutputRateCalculator28CalculateOutputRateFromRangeEN3rtc9ArrayViewIKiLln4711EEEE12native_rates' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert symbol '__ZZN6webrtc15DcSctpTransportC1EPN3rtc6ThreadEPNS1_23PacketTransportInternalEPNS_5ClockENSt3__110unique_ptrIN6dcsctp19DcSctpSocketFactoryENS8_14default_deleteISB_EEEEE14instance_count' in the debug map.
warning: (arm64)  failed to insert symbol '__ZTVNSt3__110__function6__funcIZN6dcsctp12TimerManager11CreateTimerEN4absl11string_viewENS_8functionIFNS4_8optionalINS2_10DurationMsEEEvEEERKNS2_12TimerOptionsEE3$_0NS_9allocatorISF_EEFvvEEE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZTVNSt3__110__function6__funcIZN6dcsctp15ReassemblyQueueC1EN4absl11string_viewEN6webrtc11StrongAliasINS2_6TSNTagEjEEmbE3$_0NS_9allocatorISA_EEFvN3rtc9ArrayViewIKNS2_23UnwrappedSequenceNumberIS9_EELln4711EEENS2_13DcSctpMessageEEEE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZTSNSt3__110__function6__funcIZN6dcsctp15ReassemblyQueueC1EN4absl11string_viewEN6webrtc11StrongAliasINS2_6TSNTagEjEEmbE3$_0NS_9allocatorISA_EEFvN3rtc9ArrayViewIKNS2_23UnwrappedSequenceNumberIS9_EELln4711EEENS2_13DcSctpMessageEEEE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZNSt3__1L19piecewise_constructE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZNSt3__1L19piecewise_constructE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZTVNSt3__110__function6__funcIZN6dcsctp19RetransmissionQueueC1EN4absl11string_viewEPNS2_21DcSctpSocketCallbacksEN6webrtc11StrongAliasINS2_6TSNTagEjEEmRNS2_9SendQueueENS_8functionIFvNS2_10DurationMsEEEENSE_IFvvEEERNS2_5TimerERKNS2_13DcSctpOptionsEbbE3$_0NS_9allocatorISP_EEFbNS9_INS2_14IsUnorderedTagEbEENS9_INS2_11StreamIDTagEtEENS9_INS2_6MIDTagEjEEEEE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZTSNSt3__110__function6__funcIZN6dcsctp19RetransmissionQueueC1EN4absl11string_viewEPNS2_21DcSctpSocketCallbacksEN6webrtc11StrongAliasINS2_6TSNTagEjEEmRNS2_9SendQueueENS_8functionIFvNS2_10DurationMsEEEENSE_IFvvEEERNS2_5TimerERKNS2_13DcSctpOptionsEbbE3$_0NS_9allocatorISP_EEFbNS9_INS2_14IsUnorderedTagEbEENS9_INS2_11StreamIDTagEtEENS9_INS2_6MIDTagEjEEEEE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZNSt3__1L19piecewise_constructE' in the debug map.
warning: (arm64)  failed to insert symbol '_srtp_aes_gcm_openssl_alloc' in the debug map.
warning: (arm64)  failed to insert symbol '_srtp_aes_gcm_128_openssl_description' in the debug map.
warning: (arm64)  failed to insert symbol '_srtp_aes_gcm_test_case_0' in the debug map.
warning: (arm64)  failed to insert symbol '_srtp_aes_icm_openssl_alloc' in the debug map.
warning: (arm64)  failed to insert symbol '_srtp_aes_icm_128_openssl_description' in the debug map.
warning: (arm64)  failed to insert symbol '_srtp_aes_icm_128_test_case_0' in the debug map.
warning: (arm64)  failed to insert symbol '_srtp_aes_icm_128_test_case_0_nonce' in the debug map.
warning: (arm64)  failed to insert symbol '_srtp_null_cipher_alloc' in the debug map.
warning: (arm64)  failed to insert symbol '_srtp_null_cipher_description' in the debug map.
warning: (arm64)  failed to insert symbol '_srtp_hmac_alloc' in the debug map.
warning: (arm64)  failed to insert symbol '_srtp_hmac_description' in the debug map.
warning: (arm64)  failed to insert symbol '_srtp_hmac_test_case_0' in the debug map.
warning: (arm64)  failed to insert symbol '_srtp_null_auth_alloc' in the debug map.
warning: (arm64)  failed to insert symbol '_srtp_null_auth_description' in the debug map.
warning: (arm64)  failed to insert symbol '_srtp_err_report_handler' in the debug map.
warning: (arm64)  failed to insert symbol '_bit_string' in the debug map.
warning: (arm64)  failed to insert symbol '__ZTVN3rtc16RefCountedObjectIN6webrtc12_GLOBAL__N_132IceTransportWithTransportChannelEEE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZTSN3rtc16RefCountedObjectIN6webrtc12_GLOBAL__N_132IceTransportWithTransportChannelEEE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_111g_unique_idE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZZN6webrtc19RtpReceiverInternal16GenerateUniqueIdEvE11g_unique_id' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtcL15kDtmfTonesTableE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_121kDefaultAudioSenderIdE' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_110kTransportE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_121kValidIceServiceTypesE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_115kProfileLevelIdE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN12_GLOBAL__N_126kFakeCodecFactoryCodecNameE' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_125kVp8PostProcArmFieldTrialE' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert symbol '__ZTVN6webrtc12_GLOBAL__N_112LibvpxFacadeE' in the debug map.
warning: (arm64)  failed to insert symbol '_bifilter4_coeff' in the debug map.
warning: (arm64)  failed to insert symbol '_abs_filters' in the debug map.
warning: (arm64)  failed to insert symbol '_qualify_inter_mb.ndx' in the debug map.
warning: (arm64)  failed to insert symbol '_dc_qlookup' in the debug map.
warning: (arm64)  failed to insert symbol '_once.lock' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert symbol '_pred' in the debug map.
warning: (arm64)  failed to insert symbol '_once.lock' in the debug map.
warning: (arm64)  failed to insert symbol '_kBands' in the debug map.
warning: (arm64)  failed to insert symbol '_kCat3456' in the debug map.
warning: (arm64)  failed to insert symbol '_once.lock' in the debug map.
warning: (arm64)  failed to insert symbol '_initialize_dec.init_done' in the debug map.
warning: (arm64)  failed to insert symbol '_plane_rd_mult' in the debug map.
warning: (arm64)  failed to insert symbol '_once.lock' in the debug map.
warning: (arm64)  failed to insert symbol '_dct_value_tokens' in the debug map.
warning: (arm64)  failed to insert symbol '_vp8e_usage_cfg_map' in the debug map.
warning: (arm64)  failed to insert symbol '_vp9_iht16x16_256_add_neon.IHT_16' in the debug map.
warning: (arm64)  failed to insert symbol '_sub_pel_filters_8' in the debug map.
warning: (arm64)  failed to insert symbol '_vp9_setup_mask.shift_32_y' in the debug map.
warning: (arm64)  failed to insert symbol '_mv_ref_blocks' in the debug map.
warning: (arm64)  failed to insert symbol '_dc_qlookup' in the debug map.
warning: (arm64)  failed to insert symbol '_once.lock' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert symbol '_once.lock' in the debug map.
warning: (arm64)  failed to insert symbol '_default_scan_4x4' in the debug map.
warning: (arm64)  failed to insert symbol '_seg_feature_data_max' in the debug map.
warning: (arm64)  failed to insert symbol '_mv_ref_blocks' in the debug map.
warning: (arm64)  failed to insert symbol '_once.lock' in the debug map.
warning: (arm64)  failed to insert symbol '_initialize_dec.init_done' in the debug map.
warning: (arm64)  failed to insert symbol '_inv_remap_prob.inv_map_table' in the debug map.
warning: (arm64)  failed to insert symbol '_VP9_VAR_OFFS' in the debug map.
warning: (arm64)  failed to insert symbol '_write_interp_filter.filter_to_literal' in the debug map.
warning: (arm64)  failed to insert symbol '_square' in the debug map.
warning: (arm64)  failed to insert symbol '_vp9_var_part_nnconfig_64' in the debug map.
warning: (arm64)  failed to insert symbol '_plane_rd_mult' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert symbol '_once.lock' in the debug map.
warning: (arm64)  failed to insert symbol '_quantizer_to_qindex' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert symbol '_vp9_set_rd_speed_thresholds_sub8x8.thresh_mult' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert symbol '_band_counts' in the debug map.
warning: (arm64)  failed to insert symbol '_vp9_down2_symodd_half_filter' in the debug map.
warning: (arm64)  failed to insert symbol '_best_quality_mesh_pattern' in the debug map.
warning: (arm64)  failed to insert symbol '_remap_prob.map_table' in the debug map.
warning: (arm64)  failed to insert symbol '_update_bits' in the debug map.
warning: (arm64)  failed to insert symbol '_skin_threshold' in the debug map.
warning: (arm64)  failed to insert symbol '_once.lock' in the debug map.
warning: (arm64)  failed to insert symbol '_once.lock' in the debug map.
warning: (arm64)  failed to insert symbol '_g_worker_interface' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert symbol '__ZN6webrtc12_GLOBAL__N_111kFieldTrialE' in the debug map.
warning: (arm64)  failed to insert symbol '__MergedGlobals' in the debug map.
warning: (arm64)  failed to insert

ster_onkar/Mediasoup/dependencies/libmediasoupclient/src/Consumer.cpp:3:
In file included from /Users/onkar/Downloads/mediasoup-client-swift-master_onkar/Mediasoup/dependencies/libmediasoupclient/include/Consumer.hpp:5:
In file included from /Users/onkar/Downloads/mediasoup-client-swift-master_onkar/Mediasoup/dependencies/webrtc/src/api/media_stream_interface.h:27:
In file included from /Users/onkar/Downloads/mediasoup-client-swift-master_onkar/Mediasoup/dependencies/webrtc/src/api/video/recordable_encoded_frame.h:16:
/Users/onkar/Downloads/mediasoup-client-swift-master_onkar/Mediasoup/dependencies/webrtc/src/api/units/timestamp.h:90:43: error: no member named 'IsPlusInfinity' in 'webrtc::TimeDelta'
    } else if (IsMinusInfinity() || delta.IsPlusInfinity()) {
                                    ~~~~~ ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
4 warnings and 20 errors generated.
make[2]: *** [libmediasoupclient/CMakeFiles/mediasoupclient.dir/src/Consumer.cpp.o] Error 1
make[1]: *** [libmediasoupclient/CMakeFiles/mediasoupclient.dir/all] Error 2
make: *** [all] Error 2

**Comment by lets-swapcode** [comment]
It is because Mediasoupclient and webrtc branch mentioned in the build.sh are not in sync it seems. 

Able to build it by updating the **mediasoupclient** url in the build.sh

Steps - Open build.sh and find **'function refetchLibmediasoupclient()'** and update the branch in the clone cmd. 

Either use the latest - 
git clone -b v3 --depth 1 https://github.com/VLprojects/libmediasoupclient.git

or better use the exact compatible webrtc equivalent. 
As build.sh webrtc is pointed to M112 then use -
git clone -b vl-m112.0 --depth 1 https://github.com/VLprojects/libmediasoupclient.git

And then run the build.sh in the terminal. It will work.

**Comment by fedulvtubudul** [comment]
Fixed in version 0.4.1

**Comment by 07onkar07** [comment]
worked
